File: careerforge-frontend/careerforge-frontend/src/services/interviewService.js
import axiosConfig from "../api/axiosConfig";

const interviewService = {

    getAll: () => axiosConfig.get("/interviews"),

    create: (data) => axiosConfig.post("/interviews", data),

    update: (id, data) => axiosConfig.put(`/interviews/${id}`, data),

    delete: (id) => axiosConfig.delete(`/interviews/${id}`)

};

export default interviewService;

File: careerforge-backend/src/main/java/com/careerforge/service/InterviewService.java
package com.careerforge.service;

import com.careerforge.entity.Interview;
import java.util.List;

public interface InterviewService {
    Interview add(Interview interview);
    List<Interview> getAll();
    Interview getById(Long id);
    Interview update(Long id, Interview interview);
    void delete(Long id);
}